src/triggers/default_triggers.py

```diff
@@ -32,7 +33,11 @@
             vip_dhcp_allocation=False,
             master_memory=resources.DEFAULT_MASTER_SNO_MEMORY,
             master_vcpu=resources.DEFAULT_MASTER_SNO_CPU,
-            network_type=consts.NetworkType.OVNKubernetes,
+            network_type=None,
+        ),
+        "openshift_version_4.12": Trigger(
```

Comment:
I think the name should be changed to:
```suggestion
        "let_service_choose_network_type": Trigger(
```
or something alike, because it's not specific to openshift 4.12